### PR TITLE
Updating icons strings

### PR DESCRIPTION
**Pull request by ivannaranjo**
Adding the new blue copy icon for the hover state. Updating console name.

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 1]
```diff
@@ -138,11 +138,11 @@
     <comment>Menu header for the Enable API command.</comment>
   </data>
   <data name="CloudExploreNoProjectMessage" xml:space="preserve">
-    <value>No projects found, to create a project navigate to the Google Cloud Console.</value>
+    <value>No projects found, to create a project navigate to the Cloud Platform Console.</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: The wording is a little mixed here and below.  In some places it says "... to the Cloud.." and others is just "... to Cloud..."

Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Ok, will remove the "to the" in all instances and merge.

Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: I think it should be "to the" given that Cloud Platform is a proper noun.  But I'm not the best with this sort of thing.  May be useful to talk to a tech writer.

Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: For now I am going to change it to _"No projects found, to create a project open the Cloud Platform Console"_ and will consult with the writer that is helping us.

**Pull request merged**